plugins/org.yakindu.sct.model.stext/src/org/yakindu/sct/model/stext/validation/STextJavaValidator.java

```diff
@@ -957,7 +958,7 @@ protected boolean isResponsible(Map<Object, Object> context, EObject eObject) {
 		return true;
 	}
 
-	protected Statechart getStatechart(EObject context) {
+	protected static Statechart getStatechart(EObject context) {
```

Comment:
I made this static, because it can be static and I make all my helper methods static, if it's possible.